docs/omero_figure_move.rst

```diff
@@ -0,0 +1,99 @@
+Move figures between groups
+===========================
+
+Moving figures between groups in OMERO might be necessary for example in cases where a scientist decides to move all their data into another group to enable cooperation and viewing by other colleagues or to a group which is to be made public later. This walkthrough offers two alternatives, with or without duplicating the figures and the images contained in them.
+
```

Comment:
I think it might be useful to give more background info on how figures and images work across groups. The following text might be a bit too confusing? But it would be nice if the users could understand why they're doing the steps below.

"As with all data in OMERO, Figures belong to a particular Group and are visible only to members of that Group. Figures can contain Images from the same Group as the Figure or from any other Group. Figures and the Images they contain can be independently moved from one Group to another but should ideally be kept in the same group. This avoids the situation when a user can view a Figure, but not the Images within it."

The text here mentions "two alternatives" but I see 3 walk-throughs below, so I'm unsure about the mapping. I'll read more...

Reply:
fixed in https://github.com/ome/omero-guide-figure/pull/13/commits/4eb8f01b2bd0f64dcb0ae49158bc6592b117988c